tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -90,7 +89,7 @@ def resolve_launch_arguments(args):
     # try to resolve launch file in package first
     if len(args) >= 2:
         try:
-            resolved = roslaunch.resource_cache.find_resource(args[0], args[1])
+            resolved = roslib.packages.find_resource(args[0], args[1])
```

Comment:
This function seems to be only called once. What is your use case where caching data here would improve anything?